Dockerfile

```diff
@@ -1,10 +1,12 @@
 # Base image to use at runtime
 ARG base_image=public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nonroot:2021-12-01-1638322424
 
+ARG TARGETARCH=amd64
```

Comment:
Can we name this `ARG target_arch=amd64` to keep the style consistant with the other variables?

Reply:
https://matsuand.github.io/docs.docker.jp.onthefly/engine/reference/builder/#automatic-platform-args-in-the-global-scope
<img width="966" alt="CleanShot 2023-09-27 at 09 22 15@2x" src="https://github.com/aws-controllers-k8s/code-generator/assets/22343391/5a8358b7-e0c8-4689-9474-d69c31243d71">

@a-hilaly Thank you!

TARGETARCH and target_arch in the Dockerfile are incompatible.
For example, you cannot use buildx.
Since TARGETARCH is more compatible with many container build tools, why not prioritize feature compatibility over name compatibility here?

Reply:
@a-hilaly  I'm a lover of docker such as buildx, so I'm making a suggestion. 🙇‍♂️ 
Of course I think it is an important decision to abandon compatibility with buildx. 👍 
Do you want to discard compatibility and make it a Snake Case? 👀

Reply:
@a-hilaly Do you have an opinion here?

Reply:
Trying to conform to buildx does not make sense here, since it's not used in the project. Sticking with snack case and the existing arg of`go_arch` introduces the least number of changes in the process.